### DISCUSSION TITLE
Why I can't migrate versions of apktool?

### DISCUSSION THREAD

**Discussion by elsho7** [iBotPeaches/Apktool#2655] [General]
When I installed apktool for the first time I have installed APKTOOL 2.4.3, recently, I downloaded APKTOOL 2.6 and followed steps as in the official site, yet, I seem not to be able to migrate the version. when I excute (apktool+ apktool.jar) It shows me that I am on version 2.6 but when I run command ( apktool -v) it shows me that I'm still on 2.4.3.
WHAT AM I DOING WRONG?

**Comment by iBotPeaches** [comment]
I thought I was having deja vu. I know I responded to this on the pure issue report - https://github.com/iBotPeaches/Apktool/issues/2656#issuecomment-916032168
